### ISSUE TITLE
Citizens limit increase

### ISSUE THREAD

**Issue by Evengard** [Quistar-LAB/EManagersLib#5]
I'm creating this issue as a tracking for the Citizens limit increase, as described [here](https://github.com/dymanoid/RealTime/issues/165#issuecomment-939425954).

I hope the plans for implementing it didn't go anywhere, as I think it will greatly benefit the game, even outside RealTime mod. Huge cities with low population limit are just not enjoyable as much!

**Comment by Quistar-LAB** [comment]
Increasing Citizen limit has always been a scare, because they touch pretty much every AI of the game. I would give it some thought, but my current priority is to re-write 81 tiles >> increase network (node/segment) limits >> increase building limits (this is the tough one) >> citizen limit will be considered.
I am only one person, so it'll take some time, but if more people can join and contribute, it'll probably speed things up.

**Comment by algernon-A** [comment]
Citizen Instances are indeed probably the most difficult limit to increase in the entire game, but also the next most signifcant game-limiting factor that hasn't already been addressed.

I've already done some work into increasing the limit (much, much more still to do, though), and believe that it's practically achievable, albeit a lot of work; I'm certainly indending to continue the work done so far and hope to contribute it to a future EML.  Noting that this is certainly a longer-term project.

**Comment by Quistar-LAB** [comment]
Yep, since you've already done part of it already, I feel you have a better understanding. I'd gladly accept your help in extending more limits :)
